### PR TITLE
Update docs Microsoft Graph User

### PR DESCRIPTION
**Pull request by michal-dagan**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Update docs Microsoft Graph User

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser_description.md [hunk 1]
```diff
@@ -1,30 +1,32 @@
+To use this integration, you need to configure authentication. There are three authentication methods available:  
+- Cortex XSOAR Azure app 
+- Self-Deployed Azure app  
+- Azure Managed Identities Authentication
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser_description.md:4]:
```suggestion
- [Cortex XSOAR Azure app]{#cortex-xsoar-azure-app)
- [Self-Deployed Azure app](#self-deployed-azure-app)  
- [Azure Managed Identities Authentication](#azure-managed-identities-authentication)
```

**Comment by ShirleyDenkberg** [comment]
@bziser Doc review completed.

File: Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser_description.md (hunk 1 above)
Comment (michal-dagan) [reply Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser_description.md:4]: Hi @ShirleyDenkberg, thanks for the review.
When clicking on the link in the description.md file, this window opens - https://server_url/#/cortex-xsoar-azure-app, so it is useless.

Comment (ShirleyDenkberg) [reply Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser_description.md:4]: bummer.  Works well in a readme file

**Pull request merged**